backend/src/main/java/c/team/account/UserAccountService.java

```diff
@@ -41,4 +41,8 @@ public UserAccount findByUsername(String username) {
                 .orElseThrow(() -> new UsernameNotFoundException("user does not exist: " + username));
     }
 
+    public UserAccount findByUserId(String id){
+        return Optional.ofNullable(userAccountRepository.findAccountById(id))
```

Comment:
Dzisiaj zauwazylem ze MongoRepository moze zwracac `Optional<UserAccount>`, więc można to tak przerobić

Reply:
O w sumie niezły pomysł, ale to może po Milestonie bo teraz mamy nieco ważniejsze zadania :P